modules/core/src/lib/view-manager.js

```diff
@@ -134,6 +135,26 @@ export default class ViewManager {
     return this._viewportMap[viewId];
   }
 
+  /* eslint-disable max-statements */
+  getFrustumPlanes(viewId) {
+    const {width, height} = this;
+    const view = this.getView(viewId);
+    const viewState = this.getViewState(view);
+    const viewport = view.makeViewport({width, height, viewState});
```

Comment:
Nit: This will likely create an extra viewport every frame, as we already generate viewports for rendering.